spec/views/layouts/application.html.slim_spec.rb

```diff
@@ -29,6 +29,18 @@
     end
   end
 
+  context 'shows FAKE when logging in' do
+    it 'shows banner' do
+      allow(Figaro.env).to receive(:domain_name).
+        and_return('test.login.gov')
+      allow(Rails.env).to receive(:production?).
+        and_return(true)
+
+      render
+      expect(rendered).to have_content('FAKE')
+    end
+  end
```

Comment:
What do you think about stubbing `FeatureManagement` here, since that is what the layout is calling, and it is already unit tested separately? Something like this:
```ruby
context 'when FeatureManagement.fake_banner_mode? is true' do
  it 'displays the fake banner' do
    allow(FeatureManagement).to receive(:fake_banner_mode?).and_return(true)
    render
    
    expect(rendered).to have_content('FAKE')
  end
end
```
and then the negative test as well:
```ruby
context 'when FeatureManagement.fake_banner_mode? is false' do
  it 'does not display the fake banner' do
    allow(FeatureManagement).to receive(:fake_banner_mode?).and_return(false)
    render
    
    expect(rendered).to_not have_content('FAKE')
  end
end
```

Reply:
DONE